Formula/imagemagick.rb

```diff
@@ -69,6 +71,9 @@ def install
       --enable-static
     ]
 
+    args << (build.without? "fontconfig") ? "--without-fontconfig" : "--with-fontconfig"
+    args << (build.without? "freetype") ? "--without-freetype" : "--with-freetype"
+
     if build.without? "modules"
```

Comment:
OK I was not aware of that. Let me fix it. I accidentally deleted the branch from my repository which closed the pull request :(